src/components/pages/LandingPage/index.jsx

```diff
@@ -3,13 +3,15 @@ import OjumoBriefInfo from '../../layouts/landingPage/OjumoBriefInfo'
 import MissionSection from '../../layouts/landingPage/MissionSection'
 import Footer from '../../layouts/landingPage/Footer'
 import UpcomingEvents from '../../layouts/landingPage/UpcomingEvents'
+import ArticulesSection from '../../layouts/landingPage/ArticuleSection'
```

Comment:
type on `ArticulesSection`:

```suggestion
import ArticlesSection from '../../layouts/landingPage/ArticleSection'
```